app/models/validations/shared_validations.rb

```diff
@@ -20,20 +20,16 @@ def validate_numeric_min_max(record)
       next unless question.min || question.max
       next unless record[question.id]
 
-      field = question.check_answer_label || question.id
-      min = [question.prefix, number_with_delimiter(question.min, delimiter: ","), question.suffix].join("")
-      max = [question.prefix, number_with_delimiter(question.max, delimiter: ","), question.suffix].join("")
-
       begin
         answer = Float(record.public_send("#{question.id}_before_type_cast"))
```

Comment:
I couldn't actually tell what this is doing. It doesn't stop me from being able to do the work, but I would be interested.
My best guess is that it's a funny kind of way of checking the type? We get the value before deserializing and then see if we can convert it to a float. That implies we have questions with a min max where the db type is not numeric???

Reply:
I agree with your best guess - I don't know what question this specific behaviour was needed for but yep seems like there is some question that has a numeric min/max but is cast to a string or something...